news/_posts/core-weekly/2021-11-22-coreweekly-46-2021.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 46 of 2021"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2021-11-22
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 15th to Sunday 21th of November 2021.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+[write a nice message here, or remove the "General messages" section]
```

Comment:
Please do not merge until I write a short message but I need to calm down after reading some build.prestashop.com hateful messages. I cannot write it in my current state of mind.